static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog.js

```diff
@@ -27,7 +34,10 @@ const getLiveblogEntryTimeData = el => {
     };
 };
 
-const setEpicLiveblogEntryTimeData = (el, timeData) => {
+const setEpicLiveblogEntryTimeData = (
+    el: Element,
+    timeData: TimeData
+): void => {
     if (!el) {
```

Comment:
this seems redundant with the typed parameter now - should the parameter be optional?